cfgov/unprocessed/css/cf-theme-overrides.less

```diff
@@ -144,11 +144,10 @@
    ========================================================================== */
 
 @grid_box-sizing-polyfill-path: '../js';
-@grid_wrapper-width:            1200px;
-@grid_gutter-width:             30px;
-@grid_total-columns:            12;
-@grid_debug:                    false;
-
+-@grid_wrapper-width:            1230px;
+-@grid_gutter-width:             30px;
+-@grid_total-columns:            12;
+-@grid_debug:                    false;
```

Comment:
Looks like you may have mistakenly pasted in the minus signs from the GitHub diff view of your original commit.